test/unittests/value_test.cpp

```diff
@@ -41,6 +41,12 @@ TEST(value, constructor_from_signed_ints)
     EXPECT_EQ(Value{int64_t{-3}}.i64, 0xfffffffffffffffd);
 }
 
+TEST(value, constructor_from_floating_points)
+{
+    EXPECT_EQ(Value{123.456f}.f32, 123.456f);
+    EXPECT_EQ(Value{123.456789001}.f64, 123.456789001);
```

Comment:
Would be nice to have a test for min/max.

Reply:
You mean the total max/min, or the biggest mantissa only?

Reply:
I used some special values in tests.

Reply:
Basically something which can not be represented in f32. `f64::max()` is fine, but not sure what that is actually storing, I believe a large exponent?